pkg/kubelet/kubelet_node_status.go

```diff
@@ -1049,10 +1052,13 @@ func (kl *Kubelet) setNodeStatus(node *v1.Node) {
 // defaultNodeStatusFuncs is a factory that generates the default set of
 // setNodeStatus funcs
 func (kl *Kubelet) defaultNodeStatusFuncs() []func(*v1.Node) error {
+	var mutex = &sync.Mutex{}
 	// initial set of node status update handlers, can be modified by Option's
 	withoutError := func(f func(*v1.Node)) func(*v1.Node) error {
 		return func(n *v1.Node) error {
+			mutex.Lock()
 			f(n)
+			mutex.Unlock()
```

Comment:
prefer

```
mutex.Lock()
defer mutex.Unlock()
f(n)
```

Reply:
This lock was added to fix a data race warning in the unit test. After discussing with @lantao, the data race was caused by the changes in setNodeStatusImages. Will address this issue in another PR. So remove the lock here. 
Please see the comment https://github.com/kubernetes/kubernetes/pull/60198#issuecomment-367627199